proto/firecracker.proto

```diff
@@ -54,3 +56,12 @@ message SetVMMetadataRequest {
     string VMID = 1;
     string Metadata = 2;
 }
+
+message JailerConfig {
+	// Determines whether or not the jailer should be disabled.
+	//
+	// Valid values are "ON", "OFF"
+	string State = 1;
```

Comment:
Or can we just use the lack of JailerConfig itself as a way to say disabled?

If we want to have the flag inside the field, I'd like to have `Enabled`. `JailerConfig.DisableJailer` is redundant and I think it is better to avoid negative names.

Reply:
The problem is I think we want the Jailer to be on by default (though maybe I'm mis-remembering and that wasn't part of the requirements).

That's why we may need `JailerConfig.DisableJailer` or something equivalent where the default values result in the jailer being enabled.

Reply:
> What the reason for making this a string? Can we instead have a boolean field `DisableJailer`?
because we care about unset. Since the disabling of the jailer can either happen at the config file level or the `CreateVM` call, we prioritize `CreateVM` configuration over the config file.